src/ape/api/transactions.py

```diff
@@ -40,7 +41,7 @@ class TransactionAPI(BaseInterfaceModel):
     # If left as None, will get set to the network's default required confirmations.
     required_confirmations: Optional[int] = Field(None, exclude=True)
 
-    signature: Optional[TransactionSignature] = Field(exclude=True)
+    signature: Optional[TransactionSignature] = Field(...)
```

Comment:
Could we do `Field(..., exclude=True)`.

(Also, noob question but what does `Field(...)` do here? I can't find it in the Pydantic docs)

Reply:
Field(...) just states that it's a field. No definition here at all. We had `Field(exclude=True)` before, but I need this data for the cache database and for the query. Is there another way to access it if we exclude it?

Reply:
I've seen it used as specifying some additional properties without mandating a default value

In this case, I don't think it does anything